adhocracy4/reports/serializers.py

```diff
@@ -1,9 +1,24 @@
 from rest_framework import serializers
 
+from adhocracy4.api.dates import get_datetime_display
+
 from .models import Report
 
 
 class ReportSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(required=False, source="creator.username")
+    created = serializers.SerializerMethodField(required=False)
+
+    @staticmethod
+    def get_created(report):
+        return get_datetime_display(datetime=report.created)
```

Comment:
we don't need that, see my comment above.

Reply:
oh, is it for returning the date in a specific format? if yes then it makes sense. But take the staticmethod decorator away, and replace `report` with `self`.

Reply:
done